doc/integrator/fulltext_search.rst.mako

```diff
@@ -209,6 +209,26 @@ following variables:
    default is 30.
 *  ``fulltextsearch_maxlimit`` the max possible limit, default is 200.
 
+Ranking system
+--------------
+
+By default, the fill text search uses `tsvector` and `ts_rank_cd` to rank your search results (See:
+`textsearch-controls <https://www.postgresql.org/docs/9.0/static/textsearch-controls.html>`_. These methods
+are useful to handle language-based strings. That means for instance that plural nouns are the same as
+singular nouns. This system only check if your search word exists in the result. That means that if you search
+`B 12 Zug`, `B 120 Zug` has the same weight because the system only see that the `12` exists in each case.
+
+Alternatively you can use the `similarity` system of the
+`pg_trgm module <https://www.postgresql.org/docs/9.0/static/pgtrgm.html>`. This is based only on the
+similarities of words, without language analysis, and it cares only about how near is your search with the
+result. `12` is nearer to `12` than `120`.
+To use this system, your request must contains the parameter `rank_system=similarity` and your database must
```

Comment:
How to add such parameter in the FTS request? Do this need some configuration at the client side?

Reply:
Yes, and the documentation for that will be on the client side.